recipes/sentry-native/all/conanfile.py

```diff
@@ -130,6 +130,7 @@ def validate(self):
             raise ConanInvalidConfiguration("This version doesn't support ARM compilation")
 
     def build_requirements(self):
+        self.build_requires("cmake/3.22.0")
```

Comment:
```suggestion
        if tools.Version(self.version) >= "0.4.0" and self.settings.os == "Windows":
            self.build_requires("cmake/3.22.0")
```

Reply:
Introduced in 0.4.0 and only for Windows: https://github.com/getsentry/sentry-native/blob/0.4.0/CMakeLists.txt#L1-L8